src/dom.js

```diff
@@ -377,11 +293,7 @@ var dom = module.exports = {
 				}
 				// Need to correct for borders if any exist on parent elements
 				if (node !== targetNode) {
-					if (node.currentStyle) {
-						// Oh IE, we do so love working around your incompatibilities
-						borderLeft = parseInt(node.currentStyle.borderLeftWidth, 10);
-						borderTop = parseInt(node.currentStyle.borderTopWidth, 10);
-					} else if (global.getComputedStyle) {
+					if (global.getComputedStyle) {
```

Comment:
After removing the IE8-only `currentStye` support, there are conditions to use `getComputedStyle` or a node's `style` prop value directly. I wasn't too sure about that what other platforms we support that *don't* support `getComputedStyle` either (perhaps older webOS platform?). But since I'm dealing with IE8-only, I'd rather just remove the IE8 support and leave anything else. I'd suggest removing the check in another PR, once we realize what it's there for.